### ISSUE TITLE
the puppet ticket system is so user unfriendly I'll report the problem here.

### ISSUE THREAD

**Issue by hboetes** [adrianiurca/adrianiurca.github.io#1]
Actual report from a user:

> Whenever I mis-type a cmd in the shell, I am seeing WARNING:root:could not open file '/etc/apt/sources.list'. I believe this results from Ubuntu is trying to suggest what package might provide this command. It seems /etc/apt/sources.list is no longer there. Is there a reason?

This is caused by a bug in `command-not-found`. `sources.list` being empty is not a bug.

So I disabled purge `sources.list`, but then people started adding items to it, so I created this code to ensure `sources.list` remains empty:

```
  -> exec { "echo '# ${puppet_warning}' > /etc/apt/sources.list":
    # If it contains anything else but $puppet_warning, grep will return true.
    onlyif => "grep -v '# ${puppet_warning}' /etc/apt/sources.list",
    path   => '/bin';
  }
```

Please consider changing the purge option for `sources.list` so that the file only contains a commented out warning.

Also consider switching to using GitHub bugs, since that Atlassian product is a pain.

**Comment by adrianiurca** [comment]
Hi @hboetes, I am very sad about your experience with puppet ticket system.
Is this related to the puppetlabs-apt module?

**Comment by adrianiurca** [comment]
We have each Monday Community Day, so we look to solve all problems reported by the community.
Please check the [calendar](https://puppet.com/community/office-hours/) and join us in office-hours at [slack](https://puppetcommunity.slack.com/).

kind regards,
@adrianiurca adrianiurca

**Comment by hboetes** [comment]
> Hi @hboetes, I am very sad about your experience with puppet ticket system.

Yes, it's a crime against all rules of https://developer.gnome.org/accessibility-devel-guide/stable/gad-ui-guidelines.html.en

> Is this related to the puppetlabs-apt module?

Yes, indeed.

**Comment by hboetes** [comment]
> We have each Monday Community Day, so we look to solve all problems reported by the community.
> Please check the [calendar](https://puppet.com/community/office-hours/) and join us in office-hours at [slack](https://puppetcommunity.slack.com/).

Thanks for the invitation, but I'm in CEST(+2:00) which makes for a 10-hour time difference, I'm going to bed at that time.
